### PR TITLE
[11.0.x] ISPN-12383 improved clarity in marshalling docs

### PR DESCRIPTION
**Pull request by oraNod**
Backport for https://issues.redhat.com/browse/ISPN-12383

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/proc_generating_protostream_sci.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+it.
+
+[source,java]
+----
+@AutoProtoSchemaBuilder(
+      includeClasses = SomeClass.class,
+      ...
+      service = false
+)
+----
```
Comment (ryanemerson) [inline documentation/src/main/asciidoc/topics/proc_generating_protostream_sci.adoc:74]: In `11.0.x` `SerializationContextInitializers` are not automatically registered. This behaviour was only introduced in `12.0.x`.

**Review by ryanemerson — CHANGES_REQUESTED** [review]

File: documentation/src/main/asciidoc/topics/proc_generating_protostream_sci.adoc (hunk 1 above)
Comment (oraNod) [reply documentation/src/main/asciidoc/topics/proc_generating_protostream_sci.adoc]: bah. I forgot to push some changes.

**Comment by ryanemerson** [comment]
@oraNod Looks good. Can you squash the commits and I'll merge

**Comment by oraNod** [comment]
> @oraNod Looks good. Can you squash the commits and I'll merge

@ryanemerson should be all set. thanks!

**Comment by ryanemerson** [comment]
Thanks @oraNod

**Pull request merged**